Gauge/Analysis.hs

```diff
@@ -141,6 +144,53 @@ scale f s@SampleAnalysis{..} = s {
                                , anStdDev = B.scale f anStdDev
                                }
 
+-- | The amount of time in milliseconds a benchmark must run for in order for
+-- us to have some trust in the raw measurement. We must take minimum 10
+-- samples above this threshold to perform a meaningful statistical analysis.
+threshold :: Int
+threshold = 30
```

Comment:
Untyped Int and Double ought to not be used anywhere. This really should be a better type to prevent adding different units together.

Reply:
Yes, In fact I almost made a mistake when using this.

Reply:
I am not changing it in this changeset. We should change the ones in `Measured`, this one and maybe other time quantities at the same time. All time quantities should use consistent units.